test/optim/test_optim.py

```diff
@@ -76,12 +76,19 @@ def _multistep_backprop_diff_hyperparams_fn(
 
     # This copy is necessary so the update on line 78 doesn't overwrite the original kwargs values
     kwargs = kwargs.copy()
+
+    # Have to pass in beta1 and beta2 seperately
```

Comment:
```suggestion
    # Have to pass in beta1 and beta2 separately
```
spelling nit, also below